policyengine_us/parameters/gov/states/wa/tax/income/capital_gains/deductions/standard.yaml

```diff
@@ -1,12 +1,22 @@
 description: Washington capital gains standard deduction.
 metadata:
   label: Washington capital gains standard deduction
-  name: wa_ltcg_standard_deduction
   unit: currency-USD
   period: year
   reference:
-    - title: "RCW 82.87.060(1) Deductions"
+    - title: RCW 82.87.060(1) Deductions
       href: https://app.leg.wa.gov/RCW/default.aspx?cite=82.87.060
+    - title: Capital gains tax | Washington Department of Revenue
+      href: https://dor.wa.gov/taxes-rates/other-taxes/capital-gains-tax
+    - title: 2021 Tax Legislation | Washington Department of Revenue
+      href: https://dor.wa.gov/forms-publications/publications-subject/tax-topics/2021-tax-legislation#Working%20families%20tax%20credit
+    - title: 2021 Regular Session CAPITAL GAINS TAX - Sec.7.(1)
+      href: https://lawfilesext.leg.wa.gov/biennium/2021-22/Pdf/Bills/Session%20Laws/Senate/5096-S.SL.pdf#page=9
+  uprating:
```

Comment:
This implies that the parameter is adjusted every year (with inflation) so we should have a different 2022 amount

Reply:
Should I change 2021 amount into 2022?
> Effective July 25, 2021; however, the new tax is not imposed until Jan. 1, 2022 ([ESSB 5096](http://lawfilesext.leg.wa.gov/biennium/2021-22/Pdf/Bills/Session%20Laws/Senate/5096-S.SL.pdf?q=20210517144444), Chapter 196, Laws of 2021).